frontends/.next/build-manifest.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
the `.next` folder is a build folder from nextjs. please research how they recommend handling it when it comes to checking it in to git